### DISCUSSION TITLE
Laravel Auditing Package Unicode Format Issue

### DISCUSSION THREAD

**Discussion by qalildarwish** [owen-it/laravel-auditing#784] [Q&A] [answered]
I have installed Laravel Auditing package, now when i insert or update data in Unicode format it store in database like:
{"name":**"\u0627\u062d\u0645\u062f\u0627\u0644\u0644\u0647"**,"email":"Ahmad@gmail.com","branch_id":"1","id":20}

**Comment by robin-dongbin** [comment]
Have you solve this?

**Comment by parallels999** [comment]
wich charset are you using on db?
https://github.com/owen-it/laravel-auditing/issues/530#issuecomment-511372215

**Comment by robin-dongbin** [comment]
utf8mb4

**Comment by parallels999** [comment, marked as answer]
> https://github.com/owen-it/laravel-auditing/issues/530#issuecomment-511372215

Did you try that?

**Comment by vrusua** [comment]
Hi there, did you solve this, or? Could you please share the working solution?
